### PR TITLE
updated-documentation(import.ftd)

### PR DESCRIPTION
**Pull request by arcoprova**
Updated documentation on [import page of ftd.dev](https://ftd.dev/import/)
import-before
import1-now
import-now
common-text-attributes-now

### CODE REVIEW

**Comment by amitu** [comment]
@arcoprova, the auto-import feature is not part of `ftd`. It is a `fpm` feature. So only mention the `-- import: <name>" thing here, and move the `-- auto-import:" stuff to a document in fpm.dev.

File: import.ftd [hunk 1]
```diff
@@ -7,7 +7,7 @@ id: outer
 
 -- ft.markdown:
 
-`ftd` code in one page/component gains access to the code in another page by the process of importing it. In `ftd` language the uses of `import` are two typed :
+`ftd` code in one page/component gains access to the code in another page by the process of importing it. It takes input like this :
```
Comment (amitu) [inline import.ftd]: @arcoprova the lines are too long. You have to manually put line breaks after 80 characters or so per line.

Comment (gsalunke) [reply import.ftd]: @arcoprova as advised use 80 chars limitations for all lines

Comment (arcoprova) [reply import.ftd]: ok sir

File: text-common.ftd [hunk 2]
```diff
@@ -128,7 +128,8 @@ if: not $read_more
 \-- card:
 max_lines: 1
 
-this is amplepr asd askfjkjsdfkjasdjkasd asdjaskdjaksjd kajsd kasjd askjdkasjd kasjd kasjd kasjdkasjd kasjd
+this is amplepr asd askfjkjsdfkjasdjkasd asdjaskdjaksjd kajsd kasjd askjdkasjd
```
Comment (gsalunke) [inline text-common.ftd]: @arcoprova use lorem ipsom code on such places
URL: https://www.lipsum.com

File: text-common.ftd [hunk 3]
```diff
@@ -160,4 +161,5 @@ if: not $read_more
 -- card:
 max_lines: 1
 
-this is amplepr asd askfjkjsdfkjasdjkasd asdjaskdjaksjd kajsd kasjd askjdkasjd kasjd kasjd kasjdkasjd kasjd
+this is amplepr asd askfjkjsdfkjasdjkasd asdjaskdjaksjd kajsd kasjd askjdkasjd
```
Comment (gsalunke) [inline text-common.ftd]: @arcoprova use lorem ipsom code on such places
URL: https://www.lipsum.com

File: text-common.ftd [hunk 4]
```diff
@@ -160,4 +164,8 @@ if: not $read_more
 -- card:
 max_lines: 1
 
-this is amplepr asd askfjkjsdfkjasdjkasd asdjaskdjaksjd kajsd kasjd askjdkasjd kasjd kasjd kasjdkasjd kasjd
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
+tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
```
Comment (gsalunke) [inline text-common.ftd]: @arcoprova remove few lines! this much text is not requited here...

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: text-common.ftd [hunk 5]
```diff
@@ -128,7 +128,11 @@ if: not $read_more
 \-- card:
 max_lines: 1
 
-this is amplepr asd askfjkjsdfkjasdjkasd asdjaskdjaksjd kajsd kasjd askjdkasjd kasjd kasjd kasjdkasjd kasjd
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
+tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
```
Comment (gsalunke) [inline text-common.ftd]: @arcoprova remove few lines! this much text is not requited here...

**Review by gsalunke — APPROVED** [review]

**Pull request merged**